examples/plugins/__main__.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+import logging
+
+import velum
+
+import sail
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
Can't you set the level directly in the `basicConfig` function?

Reply:
I may not remember this correctly but I'm pretty sure setting it in `basicConfig` is global and for all loggers as opposed to just the logger in this file.

Reply:
That's the point? This is an example, not a module part of the library.

Reply:
`getLogger()` without an arg also uses the global logger, so that doesn't change the behaviour either.

Reply:
Yup, though I'm probably leaving out the log setup altogether when I finish up this example (note how there's no comments explaining anything yet), as it's only there as I needed it to debug stuff.

Reply:
sorry

Reply:
Yeeted this in c5d2b41b0d46d8f0492e03fd1cbf77d72f61f59c. I'll add a bit of logging explanation later when I get around to properly documenting this example, as plugins do actually support setting per-instance loggers.